### ISSUE TITLE
[Function] Opera - Multiple file storage request during sign in and transaction

### ISSUE THREAD

**Issue by itsinz** [torusresearch/torus-website#1331]
Situation
Login on uniswap
-file storage request triggered-
click block
trigger transaction
unblock pop up
-file storage request triggered-
>> Is this intended? When does file storage request pop up?

Examples of file storage on multiple actions: Login, transact, sign
<img width="459" alt="Screenshot 2020-11-23 at 9 52 36 AM" src="https://user-images.githubusercontent.com/53069682/99923657-b075da00-2d71-11eb-8fa2-cb3b602e7a23.png">
<img width="453" alt="Screenshot 2020-11-23 at 9 49 34 AM" src="https://user-images.githubusercontent.com/53069682/99923665-b4096100-2d71-11eb-8ba2-bf14ed87242c.png">

**Comment by itsinz** [comment]
File storage keeps popping up on opera. It hinders activities because I have to keep blocking it. (I do not approve file storage)

**Comment by chaitanyapotti** [comment]
This is due to a bug in opera.
Won't fix
